src/test/java/seedu/sellsavvy/logic/commands/ordercommands/AddOrderCommandTest.java

```diff
@@ -64,7 +64,7 @@ public void execute_validPersonIndexUnfilteredList_success() {
     public void execute_duplicatePendingOrder_warningGiven() {
         AddOrderCommand addOrderCommand = new AddOrderCommand(INDEX_FIRST_PERSON, ABACUS);
         // add the first order to the person's order list
-        model.getFilteredPersonList().get(0).getOrderList().add(ABACUS);
+        model.getFilteredPersonList().get(0).getOrderList().add(ABACUS.createCopy());
```

Comment:
Why is the createCopy here needed?

Reply:
because i changed to containsDuplicateOrder in addOrder so it cannot be same item by reference